app/src/main/java/mod/hey/studios/project/proguard/ProguardHandler.java

```diff
@@ -1,6 +1,7 @@
 package mod.hey.studios.project.proguard;
 
 import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
```

Comment:
nitpick: unused import

Reply:
removed, mr nerd 🤓

Reply:
mfw not wanting warnings = nerd

Reply:
im very satisfied that you don't want one more warnin on top of hundreds (real)

Reply:
whataboutism, cool